stable/grafana/values.yaml

```diff
@@ -50,7 +50,7 @@ livenessProbe:
 
 image:
   repository: grafana/grafana
-  tag: 6.5.0
+  tag: 6.5.2
```

Comment:
please update readme to match too

Reply:
Done - noticed this hadn't been updated in a while 😅